packages/garbo/src/familiar/lib.ts

```diff
@@ -231,3 +231,9 @@ export function tcbValue(
     amuletCoin
   );
 }
+
+// 2927 knucklebones required for a TPS
+export const knuckleboneValue = () =>
+  get("_knuckleboneDrops", 0) < 100
+    ? garboValue($item`tiny plastic sword`) / 2927
+    : 0;
```

Comment:
Rather than adding a valuation for this directly in our familiar code, instead put the valuation code is garboValue and reference that.